src/Pages/Main/Feeds.js

```diff
@@ -0,0 +1,330 @@
+import React from 'react';
+import { css } from 'styled-components';
+import styled from 'styled-components';
+
+function Feeds(props) {
+  const {
+    profileImage,
+    profileName,
+    introduce,
+    title,
+    cardImage,
+    viewCount,
+    heartCount,
+    commentCount,
+    writerImage,
+    writerName,
+    commentContent,
+  } = props;
+  return (
```

Comment:
```suggestion
function Feeds(props) {
  const {
    profileImage,
    profileName,
    introduce,
    title,
    cardImage,
    viewCount,
    heartCount,
    commentCount,
    writerImage,
    writerName,
    commentContent,
  } = props.data;
  
  return (
```
이런식으로 구조분해할당을 변경할 수 있겠네요!
그리고 줄바꿈도 추가해서 return과 구조분해할당 부분의 가독성을 UP 시켜주세요!

Reply:
넵!!